LICENSE

```diff
@@ -2,11 +2,11 @@ MIT License
 
 Copyright (c) 2016 Software Engineering Education - FOSS Resources
 
-Permission is hereby granted, free of charge, to any person obtaining a copy
+Permission is hereby granted, free of charge, to any Client obtaining a copy
 of this software and associated documentation files (the "Software"), to deal
 in the Software without restriction, including without limitation the rights
 to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
-copies of the Software, and to permit persons to whom the Software is
+copies of the Software, and to permit Clients to whom the Software is
```

Comment:
I don't think we can change the copyright license?

Reply:
ohh yeahhh okay i'll change that, other than that, anything else?

Reply:
Abit hard to look through all without stepping through :/ 
Was thinking of letting it merge and we test; but see the rest

Reply:
yea to many files, but i tested with the test cases and ran the main, all seems well but 1 part says NonExistentFile.json file not found, not sure if it was there before I did all this, but i doubt so

Reply:
NonExistenceFile.json is assertFalse for every src/test/data/* folder. 
I guess it is a bug output file since the test doesn't want it to exist.

Reply:
I got the same warning and also mentioned here: https://github.com/nus-cs2103-AY2021S1/forum/issues/232 
(it's an error log output), so looks good to merge